.github/workflows/dependencies.yml

```
@@ -15,3 +15,35 @@ jobs:
       with:
         command: check
         command-arguments: all
+
+  up-to-date-plugins:
+    name: Up-to-date plugins
+    runs-on: ubuntu-latest
+    steps:
+    - name: Ensure plugins are up-to-date
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:16:8: Double quote to prevent globbing and word splitting

(shellcheck)